internal/api/fetcher/configapi.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package fetcher
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/edgelesssys/constellation/v2/internal/api/configapi"
+)
+
+// ConfigAPIFetcher fetches config API resources without authentication.
+type ConfigAPIFetcher struct {
+	*fetcher
+}
+
+// NewConfigAPIFetcher returns a new Fetcher.
+func NewConfigAPIFetcher() *ConfigAPIFetcher {
+	return &ConfigAPIFetcher{newFetcher()}
+}
+
+// FetchAzureSEVSNPVersionList fetches the version list information from the config API.
+func (f *ConfigAPIFetcher) FetchAzureSEVSNPVersionList(ctx context.Context, attestation configapi.AzureSEVSNPVersionList) (configapi.AzureSEVSNPVersionList, error) {
+	return fetch(ctx, f.httpc, attestation)
+}
+
+// FetchAzureSEVSNPVersion fetches the version information from the config API.
+func (f *ConfigAPIFetcher) FetchAzureSEVSNPVersion(ctx context.Context, attestation configapi.AzureSEVSNPVersionGet) (configapi.AzureSEVSNPVersionGet, error) {
+	// TODO check signature (sigstore.VerifySignature)
```

Comment:
```suggestion
	// TODO(elchead): check signature (sigstore.VerifySignature)
```
nit. Have a point of contact  if someone else reads the code and wants to know what needs to be done.